content/en/cloudprem/ingest_logs/observability_pipelines.md

```diff
@@ -19,42 +22,48 @@ further_reading:
+3. [**Configure the Datadog Agent**](#configure-the-datadog-agent) - Point the Agent to send logs to the deployed Worker. This step must come last because the Agent needs the Worker's address to be available.
 
 ## Create and configure an Observability Pipeline
 
-You'll need to create a pipeline that acts as an intermediary for your logs. Navigate to the [Observability Pipelines][1] page and create a pipeline using the **Log Volume Control** template. Configure the pipeline with the following components:
-- **HTTP server source**:  (No configuration) This is the entry point where the Datadog Agent will send logs. Leave the configuration empty.
-- **Empty processor**: Remove all default processors from the pipeline.
-- **HTTP client destination**: (No configuration) This forwards the logs to your CloudPrem instance. Leave the configuration empty.
-
-After configuring, your pipeline should look like this and you are ready to deploy:
+1. Navigate to [Observability Pipelines][1].
+1. Select the [**Log Volume Control template**][2].
+1. Set up your pipeline:
+    1. Choose the [**Datadog Agent** source][3].
+    1. Remove any default processors from the pipeline.
+    1. Select the [**Datadog CloudPrem** destination][4] to forward logs to your CloudPrem instance. Leave the configuration empty.
 
-{{< img src="/cloudprem/ingest/observability-pipelines-cloudprem-setup.png" alt="Screenshot of the Logs Explorer interface showing how to filter logs by selecting the cloudprem index in the facets panel" style="width:80%;" >}}
+<!-- This image shows an example with dual shipping when the instructions say log volume control -->
+<!-- {{< img src="/cloudprem/ingest/observability-pipelines-cloudprem-setup.png" alt="Screenshot of the Logs Explorer interface showing how to filter logs by selecting the cloudprem index in the facets panel" style="width:80%;" >}} -->
 
 
 ## Deploy your Observability Pipelines
 
-While other methods exist, this guide uses the Helm command for deployment. The following command installs or upgrades the pipeline, configuring it to listen for logs and forward them to your CloudPrem indexer.
+After creating your pipeline in the UI, deploy the Observability Pipelines Worker. The Worker runs your pipeline configuration and listens for logs from the Datadog Agent.
+
+The following Helm command installs or upgrades the Worker, configuring it to listen for logs and forward them to your CloudPrem indexer.
+<br>
+**Note**: You need the `pipelineId` from the pipeline you created in the previous step. This ID links the Worker to your pipeline configuration.
 
 ```shell
 helm upgrade --install opw \
 	-f values.yaml \
 	--set datadog.apiKey=XXXXXXX \
 	--set datadog.pipelineId=XXXXXXX \
-	--set env[0].name=DD_OP_SOURCE_HTTP_SERVER_ADDRESS,env[0].value='0.0.0.0:8282' \
-	--set env[1].name=DD_OP_DESTINATION_HTTP_CLIENT_URI,env[1].value='http://<RELEASE_NAME>-indexer.<NAMESPACE_NAME>.svc.cluster.local:7280/api/v2/datadog' \
```

Comment:
I wasn't sure if this needs to be removed or not? The [CloudPrem destination docs](https://docs.datadoghq.com/observability_pipelines/destinations/cloudprem/) state that 

> The Worker appends /api/v2/logs and /api/v1/validate to the endpoint URL, so these endpoints must be allowed if you are using forwarding or firewall rules.